recipe/build.sh

```diff
@@ -18,10 +18,18 @@ fi
 export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
 export CPPFLAGS="$CPPFLAGS -I$PREFIX/include"
 
+# make copy and unset CC and CXX
+# see https://github.com/conda-forge/gdal-feedstock/issues/81
+export CF_CC=$CC
+export CF_CXX=$CXX
+unset CC CXX
```

Comment:
Yeah, I had to go through some stupid stuff like this with OpenBLAS. ( https://github.com/conda-forge/staged-recipes/pull/364/commits/fed23798ddc8028a16690df848ab805ed105ad92 ) 😒